components/Landing/Header.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import classNames from 'classnames';
+import useScrollTrigger from '@material-ui/core/useScrollTrigger';
+import { AppBar, Toolbar, Button } from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+
+function ElevationScroll(props) {
+  const { children, window } = props;
+  // Note that you normally won't need to set the window ref as useScrollTrigger
+  // will default to window.
+  // This is only being set here because the demo is in an iframe.
+  const trigger = useScrollTrigger({
+    disableHysteresis: true,
+    threshold: 0,
+    target: window ? window() : undefined,
+  });
+
+  return React.cloneElement(children, {
+    elevation: trigger ? 4 : 0,
+  });
+}
+
+const Header = (props) => {
```

Comment:
Is this wip?
![image](https://user-images.githubusercontent.com/68363001/121826458-b4256b80-cc85-11eb-99ad-df606e5284f1.png)

Reply:
yes hehe